infrastructure/main.tf

```diff
@@ -96,6 +115,13 @@ module "db" {
   common_tags  = "${var.common_tags}"
 }
 
+resource "azurerm_storage_container" "document_container" {
+  name = "${var.product}-docstore-${var.env}"
+  resource_group_name = "${local.sharedResourceGroup}"
+  storage_account_name = "${local.storageAccountName}"
+  container_access_type = "private"
+}
```

Comment:
Hum, this resource is going to create preview containers in the AAT resource group and storage account. And given it's not tagged I wonder if it's going to be cleaned when the PR is closed. Though I don't see a viable alternative... We may need to keep an eye on this to see how it behaves.

Reply:
Discussed this further offline with @vlaurin we'll have `pr-NN-dm` prefixed containers inside aat. And all preview will use the store in aat.  Hopefully this gets torn down correctly.  But there wasn't an easy way to setup a `preview` shared env.